jkube-kit/generator/webapp/src/main/java/org/eclipse/jkube/generator/webapp/WebAppGenerator.java

```diff
@@ -14,33 +14,25 @@
 package org.eclipse.jkube.generator.webapp;
 
 import java.io.File;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Have you set up your IDE as per the steps provided in https://www.eclipse.org/jkube/contributing/#project-setup ? 

Usually, we avoid wildcard imports.

Reply:
> Have you set up your IDE as per the steps provided in https://www.eclipse.org/jkube/contributing/#project-setup ?
> 
> Usually, we avoid wildcard imports.

![image](https://github.com/eclipse/jkube/assets/35092323/7a0c4d6e-d2d8-4616-bd23-94a2edec7074)

This is my configuration. I followed the guide...

Reply:
Hi @ShivangMishra!
You can fix wildcard imports by following https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports